### PR TITLE
Standardization

### PR DESCRIPTION
**Pull request by MacMaru**
To enforce a consistent coding style, automate and add settings for linters and markup

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -1,29 +1,85 @@
...
-  -moz-osx-font-smoothing: grayscale;
-  text-align: center;
-  max-width: 768px;
+  @extend %typography-small;
+
   position: relative;
+  max-width: 768px;
   margin: 0 auto;
+  font-family: 'Work Sans', sans-serif;
+  font-family: 'Work Sans', sans-serif;
```
Comment (mmeester) [inline src/App.vue]: woo two font-families 🎉

File: src/components/ButtonLink.vue [hunk 2]
```diff
@@ -6,18 +6,18 @@
...
 <script>
 export default {
-  name: "ButtonLink",
+  name: 'ButtonLink',
   props: {
     link: String,
-    linktext: String
-  }
-};
+    linktext: String,
```
Comment (mmeester) [inline src/components/ButtonLink.vue]: I prefer declarative props with a `default` and maybe even a `required`

File: src/App.vue (hunk 1 above)
Comment (MacMaru) [reply src/App.vue]: Lovely!

File: src/components/ButtonLink.vue (hunk 2 above)
Comment (MacMaru) [reply src/components/ButtonLink.vue]: Indeed. I'll update linting/prettier to `vue-recommended` so this stuff will be caught

**Pull request merged**